internal/gensupport/send.go

```diff
@@ -98,10 +119,10 @@ func sendAndRetry(ctx context.Context, client *http.Client, req *http.Request, r
 		case <-ctx.Done():
 			// If we got an error, and the context has been canceled,
```

Comment:
This comment is no longer accurate, I would rephrase to something like:

```
// If we got an error and the context has been canceled, return an error acknowledging
// both the context cancelation and the service error.
```

Reply:
fixed!